daemon/daemon.go

```diff
@@ -607,7 +595,7 @@ func NewDaemon(c *Config) (*Daemon, error) {
 		consumableCache:   policy.NewConsumableCache(),
 		policy:            policy.NewPolicyRepository(),
 		ignoredContainers: make(map[string]int),
-		buildEndpointChan: make(chan *endpoint.Request, common.EndpointsPerHost),
+		buildEndpointChan: make(chan *endpoint.Request, 512),
```

Comment:
>We can have the same endpoint being rebuilt multiple times.

We can but, for example, if you try to regenerate the same endpoint 100 times in 1 second, the queue will only contain a single entry of that endpoint, not 100 entires.
The entries in the queue are guarantee to be unique.